config/jobs/kubernetes/sig-k8s-infra/trusted/sig-release-release-team-jobs/release-team-periodics.yaml

```
@@ -1,8 +1,8 @@
 periodics:
-- name: periodic-sync-enhancements-github-project-1-28
+- name: periodic-sync-enhancements-github-project-1-29
 # temporarily disable job by with an impossible cron date instead of deleting it
-  cron: '0 0 31 2 *'
-  # interval: 6h
+  # cron: '0 0 31 2 *'
+  interval: 6h
```

@npolshakova, JFYI – once we hit the "1.29 Enhancements Freeze", please pause the `periodic-sync-enhancements-github-project-1-29` job's 6-hour periodic run, with the following change. Thanks!

```suggestion
  cron: '0 0 31 2 *'
  # interval: 6h
```